src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -2,18 +2,16 @@
   <Transition name="cards" mode="out-in">
     <div v-if="playerUsername" class="player-card">
       <span class="player-name">{{ playerUsername }}</span>
-      <div class="card-container">
+      <div class="card-container" data-cy="lobby-card-container" :class="[ playerReady ? 'ready' : 'notReady' ]">
```

Comment:
`notReady` feels like a weird class name, I think it should be `not-ready`. Though it might be better to make them more deterministic like `is-ready` or `is-not-ready` if you need to. I'd also argue you may not need the not ready class at all since you would just make whatever is targeting it the default style.

Reply:
@seriouslysean you're absolutely right. I can modify that